### PR TITLE
edit blog entry to increase its readability

### PR DESCRIPTION
**Pull request by zachmandeville**

### CODE REVIEW

File: blog/content/blog/cncf-dan.md [hunk 1]
```diff
@@ -1,74 +1,196 @@
...
+relationship was formed, the servers were relocated, and are still used as part
+of the infrastructure for Hippie's vision.
+
+## Balena and Cross-Cloud
+
+Now in Aotearoa New Zealand, Hippie began doing work for [Balena](https://www.balena.io/)
+with [Denver Williams](https://github.com/denverwilliams). This technology
+enabled the "image pull and networking boot" in new, exciting ways. Inspired,
+Hippie set to use this technology with Kubernetes, by pulling down the images
+and installing [Gitlab](https://about.gitlab.com) on top of it. He showed a demo
```
Comment (BobyMCbobs) [inline blog/content/blog/cncf-dan.md:54]: Nit: _GitLab_

File: blog/content/blog/cncf-dan.md [hunk 2]
```diff
@@ -1,74 +1,196 @@
...
+they seemed ready, to share it with the community through a PR. These PR's
+invariably sparked feedback and needed revisions, and could lead to discussions
+on whether the particular test meaningfully changed the coverage at all. The
+process for writing, rewriting, and collaborating on tests was slower than
+desired.
+
+Hippie and the ii team-- notably Devan Carpenter, [Caleb
+Woodbine](https://github.com/bobymcbobs), and [Stephen
+Heywood](https://github.com/heyste)-- came up with the mock test concept, using
+org-mode in their own flavor of [Emacs (Humacs)](https://www.humacs.org/)
```
Comment (BobyMCbobs) [inline blog/content/blog/cncf-dan.md:132]: Link doesn't seem to work at this moment :/ not sure what's up

Comment (BobyMCbobs) [reply blog/content/blog/cncf-dan.md:132]: Perhaps in the mean while, use https://github.com/humacs/humacs ?

File: blog/content/blog/cncf-dan.md [hunk 3]
```diff
@@ -1,74 +1,196 @@
...
+In the interest of having a clear narrative, I edited it out.
+
+During this time Hippie was preparing a presentation about the work he was doing
+with the [Marine Reach](https://marinereach.com/) vessels. He asked Bob and his
+team if he could show them his presentation for some feedback from a live
+audience. The team really resonated with Hippie’s passion for the things he was
+involved in with Marine Reach. -->
+
+During the MTN project, Bob shipped servers to Hippie's home in Portland,
+Oregon, to help with the network-botting work. When Hippie later moved to
```
Comment (hh) [inline blog/content/blog/cncf-dan.md:39]: network-booting work

File: blog/content/blog/cncf-dan.md [hunk 4]
```diff
@@ -1,74 +1,196 @@
...
+Aotearoa New Zealand, he was unable to bring the servers with them. He asked Bob
+if he had money availble to help relocate the servers to his new home. Bob said
```
Comment (hh) [inline blog/content/blog/cncf-dan.md:41]: available

File: blog/content/blog/cncf-dan.md [hunk 5]
```diff
@@ -1,74 +1,196 @@
...
+orginzation is capable of. Hippie jumped in, helping to create a [Demo of CNCF
+technologies](https://github.com/cncf/demo/).
+
+While the demo did not get much traction, it was foundational in expoloring
+ideas which evolved into Hippie's next project with CNCF. Dan had a vision to
+get all cloud providers actively engaged in the cloud-native experience. This
+vision manifested as [Cross-Cloud](https://github.com/crosscloudci/cross-cloud),
+a way to concretely show the work of CNCF through a web
+frontend([cncf.ci](https://cncf.ci/)) that showed all the projects available on
+particpating cloud providers. [Taylor Carpenter](https://github.com/taylor), a
```
Comment (hh) [inline blog/content/blog/cncf-dan.md:66]: participating

File: blog/content/blog/cncf-dan.md (hunk 2 above)
Comment (hh) [reply blog/content/blog/cncf-dan.md:132]: I think the 'working' redirection is from http://humacs.org to https://github.com/humacs/humacs but I'm fine either way.

File: blog/content/blog/cncf-dan.md [hunk 6]
```diff
@@ -1,74 +1,196 @@
...
+meetings for initial approval, before creating the actual tests and pull
+requests.
+
+The Kubernetes project is a complex organism, with a vast community and diverse
+[Special Interest Groups](https://github.com/kubernetes/community) (SIG’s).
+Conformance cuts across all these organizational levels, and reaching consensus
+on it is an equally complex task. The collaborative test-writing method of
+Hippie’s team helped increase the velocity and transparency of test writing.
+
+<!-- Note: am not sure if this section fits fully into the flow of the essay. If
```
Comment (hh) [inline blog/content/blog/cncf-dan.md:148]: :100:

File: blog/content/blog/cncf-dan.md [hunk 7]
```diff
@@ -1,74 +1,196 @@
...
+checking and labeling of conformance pull requests, speeding up the process and
+reducing the human effort needed.
+
+## Thank you
+
+The Kubernetes Conformance journey up to this day is an eventful one, built
+around relationships and community cooperation, with many different contributors
+playing their part to help move this forward. All of this was made possible by
+the extraordinary vision and leadership of a friend dear to everyone in the
+Kubernetes community: Dan Kohn.
```
Comment (hh) [inline blog/content/blog/cncf-dan.md:194]: This is wonderful, absolutely wonderful editing Zach.
You have a gift.

**Review by hh — APPROVED** [review]
Well done. Looks good to me. Should we leave in the comments? I kinda think so.

File: blog/content/blog/cncf-dan.md (hunk 2 above)
Comment (BobyMCbobs) [reply blog/content/blog/cncf-dan.md:132]: Ah so http://humacs.org is working but not https://humacs.org

File: blog/content/blog/cncf-dan.md [hunk 8]
```diff
@@ -1,74 +1,196 @@
...
+test-writing environment. This allowed for tighter feedback loops, with test
+writers able to immediately see whether their mock test hit the endpoints they
+expected. These mock tests, along with their projected results as calculated by
+APISnoop, were then presented at the SIG Architecture Conformance sub-project
+meetings for initial approval, before creating the actual tests and pull
+requests.
+
+The Kubernetes project is a complex organism, with a vast community and diverse
+[Special Interest Groups](https://github.com/kubernetes/community) (SIG’s).
+Conformance cuts across all these organizational levels, and reaching consensus
```
Comment (mauilion) [inline blog/content/blog/cncf-dan.md:144]: s/cuts/works/ ?

File: blog/content/blog/cncf-dan.md (hunk 7 above)
Comment (mauilion) [reply blog/content/blog/cncf-dan.md:194]: great rewrite!!

File: blog/content/blog/cncf-dan.md [hunk 9]
```diff
@@ -1,74 +1,196 @@
...
+That same year, [Zach Mandeville](https://github.com/zachmandeville) joined the
+team and took over development of APISnoop. At that point there was no real
+automation for generating Snoop's data. Much was still done by manually looking
+for tests in audit event logs. Zach did a lot of the writing, rewriting, and
+architectural changes to APISnoop, along with driving updates of underlying
+Kubernetes logging so that clear coverage information could be distinguished
+from noise.
+
+While APISnoop worked to show gaps in Conformance coverage, a separate effort
+was started to fill those gaps withq tests. Unfortunately, this effort was slow
```
Comment (Riaankl) [inline blog/content/blog/cncf-dan.md:108]: gaps with tests - extra q

**Pull request merged**